### ISSUE TITLE
Redirected to localhost

### ISSUE THREAD

**Issue by lopezfelixsherwin** [forcedotcom/cli#840]
Issue Type: <b>Bug</b>

Authorized an org, redirected to the login page. Then after typing the credentials, I got redirected to this link, http://localhost:1717/OauthRedirect?code=aPrxasO.YfkYfau8m1QtGbysqdgHe_0ZHTveVWRSyTB6qFF8al6MPCAGehI_UWpUiOfkRUGZNQ%3D%3D&state=d844c2a911a3

Extension version: 50.16.0
VS Code version: Code 1.53.0 (8490d3dde47c57ba65ec40dd192d014fd2113496, 2021-02-03T20:36:38.611Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.76GB (0.55GB free)|
|Process Argv|--crash-reporter-id 9faa8645-ed60-45a0-9e6f-6166acf72147|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30211402
vstry914:30250197
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt600cf:30251589
core-portspanel:30255213

```

</details>

**Comment by mshanemc** [comment]
@lopezfelixsherwin looks like the vscode repo transferred this to CLI.  Can you open a new issue on this repo? [we have some standard questions about CLI versions] OR just look at our issue template to see how to get all those versions and add them to the existing issue?

**Comment by stest08** [comment]
Actually, this same thing is happening to me. It never returns from localhost:1717. Proxy environment variables are set and this has never been an issue before. 

**sfdx-cli --version** 
- sfdx-cli/7.85.1-2fb9e41053 darwin-x64 node-v12.18.3

**sfdx plugins --core**
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.1 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.5 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.5 (core)
analytics 1.12.1 (core)
auth 1.4.6 (core)
config 1.2.3 (core)
generator 1.1.3 (core)
salesforcedx 50.15.0 (core)
├─ schema 1.0.3 (core)
├─ user 1.0.9 (core)
├─ limits 1.0.3 (core)
├─ custom-metadata 1.0.11 (core)
├─ apex 0.1.4 (core)
├─ templates 50.4.0 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
└─ salesforce-alm 50.15.0 (core)
sfdx-cli 7.85.1 (core)
